src/Matcher/Pattern/RegexConverter.php

```diff
@@ -21,6 +21,8 @@ public function toRegex(TypePattern $typePattern) : string
                 return '(\\-?[0-9]*)';
             case 'double':
                 return '(\\-?[0-9]*[\\.|\\,][0-9]*)';
+            case 'uuid':
+                return "([\da-f]{8}-[\da-f]{4}-[1-5][\da-f]{3}-[89ab][\da-f]{3}-[\da-f]{12})";
```

Comment:
I think we already have this pattern somewhere, maybe in UuidMatcher? Could you try to reuse it without duplicatuin?

Reply:
actually to avoid BC break we would need to have 3 consts. Those 2 that you mentioned (with different names) above and 1 with not changed name builded from those 2. 
However I don't believe anybody would use this pattern anywhere outside php-matcher, but still it's better to be safe then sorry

Reply:
I pushed a new commit with two constants : 

```
const UUID_PATTERN = '[\da-f]{8}-[\da-f]{4}-[1-5][\da-f]{3}-[89ab][\da-f]{3}-[\da-f]{12}';
const UUID_FORMAT_PATTERN = '|^'.self::UUID_PATTERN.'$|';
```